File: ProjetoIgreja/Back/src/IgrejaJdLilah.Domain/Entidades/Contato.cs
using System;

namespace IgrejaJdLilah.Domain.Entidades
{
    public class Contato
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Email { get; set; }
        public string Telefone { get; set; }
        public string Denominacao { get; set; }
        public DateTime DataCadastro { get; set; }
        public bool FazParteMesmaDenominacao { get; set; }
        public int EnderecoId { get; set; }
        public Endereco Endereco { get; set; }
    }
}

File: ProjetoIgreja/Front/IgrejaJdLilah-APP/src/app/models/Contato.ts
export interface Contato {
   id: number;
   nome: string;
   email: string;
   telefone: string;
   denominacao: string;
   dataCadastro: Date;
   fazParteMesmaDenominacao: boolean;
   enderecoId: number;
   endereco: string;

}
